sources/core/Stride.Core.Mathematics/Vector3.cs

```diff
@@ -239,6 +239,34 @@ public void Pow(float exponent)
             Z = MathF.Pow(Z, exponent);
         }
 
+        /// <summary>
+        /// Rotates the vector around the origin by the angle using the given axis of rotation.
+        /// </summary>
+        /// <param name="axis">The axis of rotation.</param>
+        /// <param name="angle">The angle of rotation.</param>
+        public void RotateAround(Vector3 axis, float angle)
+        {
+            RotateAround(Zero, axis, angle);
+        }
+
+        /// <summary>
+        /// Rotates the vector around the target by the angle using the given axis of rotation.
+        /// </summary>
+        /// <param name="target">The position vector of the target to rotate around.</param>
+        /// <param name="axis">The axis of rotation.</param>
+        /// <param name="angle">The angle of rotation.</param>
+        public void RotateAround(Vector3 target, Vector3 axis, float angle)
```

Comment:
I think we should be as consistent as possible and avoid code duplication; since most of the functions here are statics let's keep the static one below and get rid of those two.

I understand that you might want instance methods for succinctness, if that's the case you can work around this by adding a ``using static Stride.Core.Mathematics.Vector3;`` to your class which would give you the ability to omit the type when calling static methods of that type. So you would be able to write ``Vector3.Lerp(x, y, z)`` as just ``Lerp(x, y, z)``.

Reply:
Done!